components/security/authentication.rst

```diff
@@ -314,6 +315,9 @@ order to give your user a welcome flash message every time they log in.
 The ``security.switch_user`` event is triggered every time you activate
 the ``switch_user`` firewall listener.
 
+The ``security.logout_on_change`` event is triggered when a token has been deauthenticated
+because of a user change, it can help you doing some clean-up task when a logout has been triggered.
```

Comment:
Please add a versionadded directive here about the new event

Reply:
done